### PR TITLE
Do not discover unregistered API #5

### PR DESCRIPTION
**Pull request by rathnapandi**
Fix to discover active apis

### CODE REVIEW

**Comment by rathnapandi** [comment]
change filtering based on querystring

**Review by tsjohns9 — COMMENTED** [review]
This looks good. Can you show it to me? I will approve it after that.

File: pkg/anypoint/types.go
Comment (tsjohns9) [inline pkg/anypoint/types.go:95]: Is this field still needed after your new changes?

File: pkg/anypoint/client.go
Comment (tsjohns9) [inline pkg/anypoint/client.go:466]: Please remove the commented code before merging

File: pkg/anypoint/types.go
Comment (rathnapandi) [reply pkg/anypoint/types.go:95]: No, i can remove it.

File: pkg/discovery/servicehandler.go
Comment (rathnapandi) [reply pkg/discovery/servicehandler.go]: agree

**Comment by rathnapandi** [comment]
Fix review comments

**Review by tsjohns9 — APPROVED** [review]

**Pull request merged**
